app/controllers/admin/npq/applications/notes_controller.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+module Admin
+  module NPQ
+    module Applications
+      class NotesController < Admin::BaseController
+        skip_after_action :verify_authorized
+        skip_after_action :verify_policy_scoped
+
+        def edit
+          @npq_application = NPQApplication.find(params[:id])
+        end
+
+        def update
+          npq_application = NPQApplication.find(params[:id])
+          npq_application.assign_attributes(note_params)
+
+          if npq_application.save
+            name = npq_application.participant_identity.user.full_name
+
+            flash[:success] = {
+              title: "#{name} updated",
+              content: "#{name} has a new note: '#{npq_application.notes.humanize.downcase}' ",
```

Comment:
For particularly long notes this can get weird:

<img width="1784" alt="Screenshot 2023-03-21 at 14 24 49" src="https://user-images.githubusercontent.com/1465268/226636749-6f64ab0f-eed4-447a-b6e6-4b8f8b8f07f9.png">

We may want to just say "#{name} has a new note, see below for details." rather than inserting the entire note into the flash.